### PR TITLE
Rename to Tahoma2D

### PR DESCRIPTION
**Pull request by jeremybullock**
This renames the program to Tahoma2D.

@manongjohn If you wouldn't mind, I'd love a quick glance over at this.  
Especially on the build scripts.

### CODE REVIEW

**Comment by manongjohn** [comment]
Sure.  I'll take a look at it tonight

**Comment by jeremybullock** [comment]
Thanks man.

File: toonz/sources/toonz/CMakeLists.txt [hunk 1]
```diff
@@ -383,11 +383,11 @@ endif()
 qt5_wrap_cpp(SOURCES ${MOC_HEADERS})
 
 if(BUILD_ENV_MSVC)
-    add_executable(Tahoma WIN32 ${HEADERS} ${SOURCES} ${OBJCSOURCES} ${RESOURCES} ${RC_FILE})
+    add_executable(Tahoma2D WIN32 ${HEADERS} ${SOURCES} ${OBJCSOURCES} ${RESOURCES} ${RC_FILE})
 else()
     set(MACOSX_BUNDLE_ICON_FILE "Tahoma.icns")
     set_source_files_properties(Tahoma.icns PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
-    add_executable(Tahoma MACOSX_BUNDLE Tahoma.icns ${HEADERS} ${SOURCES} ${OBJCSOURCES} ${RESOURCES})
+    add_executable(Tahoma2D MACOSX_BUNDLE Tahoma.icns ${HEADERS} ${SOURCES} ${OBJCSOURCES} ${RESOURCES})
```
Comment (manongjohn) [inline toonz/sources/toonz/CMakeLists.txt:390]: Suggestion: Change `Tahoma.icns` to `Tahoma2D.icns` in the 3 lines above and also rename the file.

**Comment by jeremybullock** [comment]
Thanks @manongjohn for the thorough review.  I think I took care of everything.  I appreciate your help.

**Comment by jeremybullock** [comment]
I'll leave this to you to merge.

**Comment by manongjohn** [comment]
Will merge after doing quick build/package tests, just to make sure we didn't miss something.

**Comment by manongjohn** [comment]
Might want to update this graphic on the About screen:
image

**Comment by jeremybullock** [comment]
I'll take care of that one later.  Thanks for pointing it out.

**Comment by jeremybullock** [comment]
I updated the graphics.

**Review by manongjohn — APPROVED** [review]

File: stuff/profiles/layouts/fxs/fxs.lst [hunk 2]
```diff
@@ -162,7 +162,7 @@
     STD_posterizeFx
     STD_solarizeFx 
   </Stylize>
-  <Tahoma_Level>
+  <Tahoma2D_Level>
```
[the remark this answers is not in the document]
Comment (jeremybullock) [reply stuff/profiles/layouts/fxs/fxs.lst:165]: I understand. Color mapped? Color indexed? Magic?

**Comment by jeremybullock** [comment]
@manongjohn Is this ready to be merged?  I think the fx category can continue to be discussed, but is an easy enough fix for a later pr.

**Comment by manongjohn** [comment]
Yes. Was just about to merge it :)

Suggestions for users with existing build environments after pulling in these changes:

- Windows builds: After you compile using VS, it will generate a new `Tahoma2D.sln` project file automatically which you should use going forward. Any customizations you made to the project that would be stored in `Tahoma.sln` would need to be redone.  Delete existing build `Tahoma.exe` artifacts. 
- Linux builds:  Delete your toonz/build directory and rebuild.
-  macOS builds (command line): I didn't have to do anything. I was able to just run the build scripts again. If you run into trouble, I do suggest deleting the toonz/build directory and try again
- macOS builds (Xcode): I didn't try this, though typically changes like this for me in the past would have required me to redo the 'cmake' step and create a new project file.

**Comment by jeremybullock** [comment]
Thanks @manongjohn I really appreciate your help with this.

**Pull request merged**